direct/io-blob/src/main/java/cz/o2/proxima/direct/blob/BulkBlobWriter.java

```diff
@@ -44,7 +46,7 @@
 
   @Getter private final AccessorT accessor;
   @Getter private final Context context;
-  private transient BulkAttributeWriter wrap;
+  private transient @Nullable BulkAttributeWriter wrap;
```

Comment:
under what circumstances is this set to null?

Reply:
before call to `init`, which is called in write only

Reply:
👍